pwa/src/components/PatientInfoForm.tsx

```diff
@@ -0,0 +1,265 @@
+      />
+
+      <List renderHeader={() => "Sex"}>
+        {sexValues.map((i) => (
+          <RadioItem
+            data-testid={`${i.label.toLowerCase()}-button`}
+            id={`${i.label.toLowerCase()}-button`}
+            key={i.value}
+            name="sex"
+            checked={sex === i.value}
+            onClick={() => setSex(i.value)}
+          >
+            {i.label}
+          </RadioItem>
+        ))}
+      </List>
+
+      <WhiteSpace />
+
+      <InputItem placeholder="YYYY-MM-DD" id="birthdate">
+        Birth Date
+      </InputItem>
+
+      <InputItem type="number" placeholder="Age" id="age">
+        Age{" "}
+      </InputItem>
+
+      <InputItem type="number" placeholder="Phone" id="phone">
+        Phone
+      </InputItem>
+
+      <TextareaItem title="Town" placeholder="Town" id="town" autoHeight />
+
+      <List renderHeader={() => "Visit?"}>
+        {radioYesNo.map((i) => (
+          <RadioItem
+            key={i.value}
+            data-testid={`visit-${i.label}`}
+            name="visit"
+            checked={visit === i.value}
+            onClick={() => setVisit(i.value)}
+          >
+            {i.label}
+          </RadioItem>
+        ))}
+      </List>
+
+      {/* Only display the 'female-only' div if the patient is female */}
+      {sex ? (
+        <div id="female-only">
+          <WhiteSpace />
+          <List renderHeader={() => "Pregnant?"}>
+            {radioYesNo.map((i) => (
+              <RadioItem
+                data-testid={`pregnant-${i.label}`}
+                key={i.value}
+                name="pregnant"
+                checked={pregnant === i.value}
+                onClick={() => setPregnant(i.value)}
+              >
+                {i.label}
+              </RadioItem>
+            ))}
+          </List>
+
+          <InputItem title="G" type="number" id="gravida">
+            G
+          </InputItem>
+
+          <InputItem title="P" type="number" id="para">
+            P
+          </InputItem>
+
+          <InputItem title="A" type="number" id="abortus">
+            A
+          </InputItem>
+
+          <TextareaItem title="LMP" id="lmp" />
+
+          <List renderHeader={() => "Wants Planning?"}>
+            {radioYesNo.map((i) => (
+              <RadioItem
+                data-testid={`planning-${i.label}`}
+                key={i.value}
+                name="planning"
+                checked={planning === i.value}
+                onClick={() => setPlanning(i.value)}
+              >
+                {i.label}
+              </RadioItem>
+            ))}
+          </List>
+        </div>
+      ) : null}
+
+      <WhiteSpace />
+
+      <InputItem type="number" placeholder="Weight" id="weight">
+        Weight (kg)
+      </InputItem>
+
+      <InputItem type="number" placeholder="Height" id="height">
+        Height (cm)
+      </InputItem>
+
+      <InputItem type="number" placeholder="Blood Pressure" id="bp">
+        BP
+      </InputItem>
```

Comment:
Blood pressure need two measurements, or two allow the insertion of the `/` character.
BP is measure as a fraction. (systolic / diastolic) example `100/50`
currently, this is a single number input field

Reply:
Fixed [in this commit](https://github.com/Haiti-Distributed-Healthcare-System/hdhs/pull/66/commits/465c99790062ffef7f669e472a06efcdf65969ca)